server/jira.js

```diff
@@ -257,17 +259,58 @@ Jira.verifyTicketOnDev = function(ticket) {
     return;
   }
 
-  var jiraId = ticket.jiraId
-  var url =  "issue/" + jiraId + "/transitions"
+  var jiraId = ticket.jiraId;
+  var url =  "issue/" + jiraId + "/transitions";
+  // Perform transition id 131, aka "Verify" action
   var data = {
     transition: {
       id: 131
     }
-  }
+  };
 
   Jira.makePostRequest(url, data);
 };
 
+Jira.reOpenTicket = function(ticket, failReason) {
```

Comment:
This method should only be concerned with re opening the ticket, so let's remove the commenting step:
```js
Jira.reOpenTicket = function(ticket) {
  // Don't re-open ticket if it's already re-opened (in the to do state). 
  if (ticket.statusName !== Config.jira.toDoStatusName) {
    var jiraId = ticket.jiraId;
    var url =  "issue/" + jiraId + "/transitions";
    // Perform transition id 201, aka "Reopen" action
    var data = {
      transition: {
        id: 201
      }
    };

    Jira.makePostRequest(url, data);
  }
};

Jira.commentTicket = function(ticket, failReason) {
  var comments = "Failed by wrangler. ";

  if (!failReason) {
    comments += "Tester did not provide comments.";
  } else {
    comments += "Tester's comments: " + failReason;
  }

  var jiraId = ticket.jiraId;
  var url = "issue/" + jiraId + "/comment";
  var data = {
    body: comment
  };

  Jira.makePostRequest(url, data);
}
```